cli/compose/convert/service.go

```diff
@@ -46,6 +46,12 @@ func Services(
 		if err != nil {
 			return nil, errors.Wrapf(err, "service %s", service.Name)
 		}
+		// add an image label to serviceSpec
+		if serviceSpec.Labels == nil {
+			serviceSpec.Labels = make(map[string]string)
+		}
+		serviceSpec.Labels["com.docker.stack.image"] = service.Image
```

Comment:
We already add labels in `convertService()`, we should probably keep this together with the other labels we're adding.

A constant for this label would be good as well.